src/platform/qt/library/LibraryController.h

```diff
@@ -0,0 +1,105 @@
+/* Copyright (c) 2014-2017 waddlesplash
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#pragma once
```

Comment:
`#pragma once` isn't used anywhere in this codebase. I'm not opposed to revamping the entire Qt frontend to use it, but maintaining consistency until such time is a good idea. Cf. my stance of signals/slots.